docs/maui/TOC.yml

```
@@ -119,6 +119,10 @@ items:
       href: converters/string-to-list-converter.md
     - name: "TextCaseConverter"
       href: converters/text-case-converter.md
+- name: "Layouts"
+  items:
+    - name: "UniformItemsLayout"
+      href: layouts/UniformItemsLayout.md
```

Add index to the TOC

```suggestion
- name: "Layouts"
  items:
    - name: Overview
      href: layouts/index.md
    - name: "UniformItemsLayout"
      href: layouts/UniformItemsLayout.md
```